binance/spot/convert.py

```diff
@@ -0,0 +1,14 @@
+from binance.lib.utils import check_required_parameters
+
+
+def my_converts(self, startTime, endTime, **kwargs):
```

Comment:
- Please consider method name `convert_trade_history`, being more detailed can provide clear meaning of the method.
- Please also add the method docstring, you can take example from other modules' methods. It's used to generate documentation in https://binance-connector.readthedocs.io/en/stable/.

Reply:
Done, detailed answer below.